docs/rst/statistics_backend/get_graph.rst

```diff
@@ -0,0 +1,68 @@
+.. include:: ../exports/alias.include
+
+.. _statistics_backend_get_graph:
+
+Get entities' graph
+-------------------
+
+.. warning::
+    This feature is currently not supported.
+    It will be implemented on a future release of *Fast DDS Statistics Backend*.
+
+*Fast DDS Statistics Backend* allows to retrieve the entire graph of entities for which the singleton holds statistics
+data.
+The result of this query is a |Graph-api| tree structure that contains the |EntityId-api| and names of each entity.
+To be able to understand and interpret this tree, it is required to know about all the available entities and the inner
+relations between them.
+Following, there is a diagram of the relation between the *Fast DDS Statistics Backend*, and how are they divided into
+physical and domain related.
+For more information about the different |EntityKind-api| please refer to :ref:`types_entity_kind`.
+
+.. figure:: /rst/figures/internal_db.svg
+    :align: center
+
+    *Fast DDS Statistics Backend* entity relations and their division into physical and domain related.
+
+For the following example, a simple scenario is considered, where there is one process running two participants on the
+same domain; one with a data reader and the other one with a data writer (both in the same topic).
+This means that there is only one |USER-api| within a single |HOST-api|.
+The application can retrieve the network graph by:
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+    :language: c++
+    :start-after: //CONF-GET-GRAPH-EXAMPLE
+    :end-before: //!
+    :dedent: 8
+
+In this example, the previous call would return a |Graph-api| object similar to the following:
+
+.. literalinclude:: /code/graph_example.json
```

Comment:
I thought we would remove metainformation from the graph, please recheck this design point.

Reply:
I think we will, but I rather not modify the example until we get to the actual implementation